test/e2e/util.go

```diff
@@ -2193,7 +2193,7 @@ func restartKubeProxy(host string) error {
 	if !providerIs("gce", "gke", "aws") {
 		return fmt.Errorf("unsupported provider: %s", testContext.Provider)
 	}
-	_, _, code, err := SSH("sudo /etc/init.d/kube-proxy restart", host, testContext.Provider)
```

Comment:
also, a minor suggestion (maybe to consider along with a larger cleanup of `SSH` calls): including stderr in the error message probably would've made tracking down this failure much easier (rather than just getting the cryptic "code 1" error message).